website/videoBlog/2023-11-21-RFC-14-Step-by-Step-Guide-for-Incremental-Data-Pull-from-Postgres-to-Hudi-using-deltastreamer.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "RFC-14: Step-by-Step Guide for Incremental Data Pull from Postgres to Hudi using DeltaStreamer (#4)"
+last_modified_at: 2023-12-20T16:54:38.964863-07:00
+authors:
+- name: Soumil Shah
+category: blog
+image: /assets/images/video_blogs/2023-11-21-RFC-14-Step-by-Step-Guide-for-Incremental-Data-Pull-from-Postgres-to-Hudi-using-deltastreamer.png
+navigate: "https://www.youtube.com/watch?v=kqQ0SVwfBig"
+tags:
+- guide
+- beginner
+- deltastreamer
```

Comment:
Lets remember to add hudi streamer tag wherever we are adding deltastreamer. For reference deltastreamer was renamed in recent releases.